src/core/CoolBoardSensors.cpp

```diff
@@ -57,11 +57,11 @@ void CoolBoardSensors::begin() {
   }
   this->setEnvSensorSettings();
   delay(100);
-  uint8_t envSensorStatus = this->envSensor.begin();
+  this->envSensor.begin();
   // Make sure sensors had enough time to turn on.
   // BME280 requires 2ms to start up.
   delay(1000);
-  DEBUG_VAR("BME280 status after begin is:", envSensorStatus);
+  DEBUG_VAR("BME280 status after begin is:", this->envSensor.begin());
```

Comment:
Ah, I don't think you should call `begin()` twice, sorry... What was the warning here ?

Reply:
is a warning when you compile in -e prod, 
he say envSensorStatus is a unused variable

Reply:
No, we can't merge that and we need to find another solution to fix this. Isn't there a `sensor.getStatus()` method available on the BME class ?